libnavui-dropin/src/main/java/com/mapbox/navigation/dropin/internal/extensions/MapboxExtendableButtonEx.kt

```diff
@@ -0,0 +1,22 @@
+package com.mapbox.navigation.dropin.internal.extensions
```

Comment:
@file:JvmName("MapboxExtendableButtonEx"). Same applies to any other extensions you have created in this PR. Eg: https://github.com/mapbox/mapbox-navigation-android/pull/6273/files#r969210459